subsys/bluetooth/mesh/provisioner.c

```diff
@@ -317,12 +317,14 @@ static void send_confirm(void)
 
 	if (bt_mesh_prov_conf(bt_mesh_prov_link.conf_key,
 			      bt_mesh_prov_link.rand, bt_mesh_prov_link.auth,
-			      net_buf_simple_add(&cfm, 16))) {
+			      bt_mesh_prov_link.conf_local)) {
 		BT_ERR("Unable to generate confirmation value");
 		prov_fail(PROV_ERR_UNEXP_ERR);
 		return;
 	}
 
+	memcpy(net_buf_simple_add(&cfm, 16), bt_mesh_prov_link.conf_local, 16);
```

Comment:
```suggestion
	net_buf_simple_add_mem(&cfm, bt_mesh_prov_link.conf_local, 16);
```

Reply:
Yeah, I saw it just as I posted it :smile: